stream/statelib/src/test/java/org/apache/bookkeeper/statelib/impl/kv/TestStateStore.java

```diff
@@ -121,6 +123,18 @@ public void enableCheckpoints(boolean enable) {
         }
     }
 
+    public void setRemoveLocal(boolean enable) {
+        removeLocal = enable;
```

Comment:
Having this removeLocal and removeRemote to be mutable will not change the state of internal instance of `RocksdbCheckpointTask` which are passed these variables during the init function call, right?

How is the removeLocal and removeRemote variables used beside being passed to the constructor of `RocksdbCheckpointTask` ?

Reply:
These variables are only used to initialize the `RocksdbCheckpointTask`. These are mutable to allow use to re-initialize the statestore instance with different options while testing. Also this is only used in test cases.

Reply:
Ah, yes the `init()` is also called from `restore()` function